standards/application/privatev1.md

```diff
@@ -0,0 +1,417 @@
+    C --> S --> R --> E --> D
+
+    classDef plain fill:none,stroke:transparent;
+```
+**Pipeline Stages:**
+- **Segmentation**: Divides content into transport-appropriate fragments
+- **Reliability (SDS)**: Adds tracking metadata for delivery detection and ordering
+- **Encryption (Double Ratchet)**: Provides confidentiality, authentication, and forward secrecy
+
+
+
+### Segmentation
+
+While PRIVATE1 itself has no inherent message size limitation, practical transport mechanisms typically impose maximum payload sizes.
+
+**Why Segment Before Encryption**
+
+Segmenting after encryption would force the transport layer to handle fragmentation of ciphertext blobs, creating several problems.
+- Transport-layer segmentation would require buffering all segments before any can be authenticated, increasing the DOS attack surface.
+- Unauthenticated segment reassembly opens the door to malicious segment injection and substitution attacks.
+- Unencrypted segmentation metadata reveals size and other metadata about the content in transit. 
+
+**Why Segment Before Reliability**
+
+Placing segmentation after reliability tracking would mean retransmission of a dropped segment would require re-broadcasting the entire frame.
+By segmenting first, the reliability layer can track individual segments and request retransmission of only the missing fragments.
+
+**Implementation**
+
+The segmentation strategy used is defined by [!TODO: Flatten link once completed](https://github.com/waku-org/specs/pull/91)
+
+Implementation specifics:
+- Error correction is not used, as reliable delivery is already provided by lower layers. 
+- `segmentSize` = `max_seg_size`
+- All payloads regardless of size are wrapped in a segmentation message.
+ 
+### Message Reliability
+
+Scalable Data Sync (SDS) is used to detect missing messages, provide delivery confirmation, and handle retransmission of payloads.
+SDS is implemented according to the [specification](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/sds.md).
+
+**SDS Field Mappings**
+
+The following mappings connect PRIVATE1 concepts to SDS fields:
+
+- `sender_id`: !TODO: This requires PRIVATE1 to be identity aware
+- `message_id`: uses the `frame_id` definition. 
+- `channel_id`: uses the `conversation_id` parameter.
+
+**Sender Validation**
+SDS uses a `sender_id` payload field to determine whether a message was sent by the remote party. This value is sender reported and not validated which can have unknown implications if trusted in other contexts. For security hygiene Clients SHOULD drop SDS messages if `sender_id` != the sender derived from the encryption layer. !TODO: PRIVATE1 is not sender aware currently
+
+**Bloom Filter Configuration**
+
+PRIVATE1 uses bloom filter parameters of `n=2000` (expected elements) and `p=0.001` (false positive probability).
+This configuration produces bloom filters of approximately 3.5 KiB per message.
+
+!TODO: Can the bloom filter be dropped in 1:1 communication?
```

Comment:
I agree with the split.
Only use casual history is enough for message sent check.